README.md

```diff
@@ -70,6 +70,10 @@ The `yarn build` step will download the latest TinyMCE package and generate new
 
 The TinyMCE API documentation is maintained within the [TinyMCE project repository](https://github.com/tinymce/tinymce) and compiled for the documentation site using [MoxieDoc](https://github.com/tinymce/moxiedoc).
 
+To update the published API documentation, change the version in the `.api-version` file, run `yarn build` and then commit the changes. Running `yarn build` will download the TinyMCE package specified in `.api-version` and generate new API reference content from source code.
+
+**Note:** The API documentation should never be edited manually.
```

Comment:
```suggestion
NOTE: The API documentation should never be edited manually.
```

*NOTE:* is an [admonition block](https://docs.asciidoctor.org/asciidoc/latest/blocks/admonitions/). It doesn’t need to be marked-up for presentation purposes, our CSS will do that for us.

Reply:
@brian-forte this is **Markdown** not AsciiDoc 😉